### PR TITLE
refactor: clean up sqitch plan

### PR DESCRIPTION
**Pull request by matthieu-foucault**

### CODE REVIEW

**Comment by matthieu-foucault** [comment]
@wenzowski , @AntBush , @marcellmueller , @VB-at-Bis , can you all review and either approve or request changes to the documentation that is added in this PR?

**Review by marcellmueller — COMMENTED** [review]
Looks good but I think you need to rebuild the graphql schema since the diff is failing.

File: db/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+- `schemas`: we shouldn't need to add more than the public and private schemas
+- `tables`: all tables go in there
+- `types`: all composite or enum types go in there.
+- `functions`: for functions that would result in a [postgraphile custom query](https://www.graphile.org/postgraphile/custom-queries/)
+- `computed_columns`: for functions that would result in a [postgraphile computed column](https://www.graphile.org/postgraphile/computed-columns/)
+- `mutations`: for [custom mutations](https://www.graphile.org/postgraphile/custom-mutations/)
+- `trigger_functions`: for any function returning a trigger
+- `util_functions`: for any function used primarily to increase code reuse, either in the public or private schema
+- `views`: all views go in there
```
Comment (wenzowski) [inline db/README.md:21]: I really like the pattern of splitting out functions based on how they're going to be used.

**Review by wenzowski — COMMENTED** [review]
Documentation is crystal clear. Love it!

**Review by wenzowski — APPROVED** [review]

**Pull request merged**